main.py

```diff
@@ -1,12 +1,245 @@
+        if video_id.isnumeric():
+            video_id = int(video_id)
+            video = VideoAPI.delete_video(video_id)
+            if not video:
+                print(f"{Fore.RED}Video with this ID does not exist.")
+            print(f"{Fore.GREEN}Video successfully deleted.")   
+
+    elif choice == '4':
+        all_videos = [video for video in VideoAPI.list_videos()]
+        for i in tqdm(all_videos, colour='green'):
+            time.sleep(.01)
+        for video in all_videos:
+            index =all_videos.index(video)
+            print(f'\n{index+1}. Title: {video["title"]} | ID: {video["id"]} | Release Date: {video["release_date"]} | Total Inventory #:{video["total_inventory"]}')
+    
+    elif choice=='5':
+        print(f"{Style.BRIGHT}\nEnter VIDEO ID to view video info.")
+        video_id=input('Video ID:')
+        while video_id.isalpha() or not video_id:
+            video_id = input(f"{Fore.RED}Could not select. Please enter a numerical ID:")
+        if video_id.isnumeric():
+            video_id = int(video_id)
+            video = VideoAPI.get_video(video_id)
+            if not video:
+                print(f"{Fore.RED}Video with this ID does not exist.")
+            print(f'\nTitle: {video["title"]} | ID: {video["id"]} | Release Date: {video["release_date"]} | Total Inventory #:{video["total_inventory"]}')
+
+
+############################################################################################
+
+def rental_endpoints(choice):
+    if choice=='11':
+        print(f"{Style.BRIGHT}\nChecking out? Whatcha got?")
+        user_id=input('User ID: ')
+        video_id=input('Video ID: ')
+        while video_id.isalpha() or user_id.isalpha() or not video_id or not user_id:
+            print(f"{Fore.RED}User ID or Video ID is not valid. Please enter a valid number.")
+            user_id=input('User ID: ')
+            video_id=input('Video ID: ')
+        if user_id.isnumeric():
+            user_id = int(user_id)
+        if video_id.isnumeric():
+            video_id = int(video_id)
+        RentalAPI.checkout(user_id=user_id, video_id=video_id)
+        print(f"{Fore.GREEN}Video #{video_id} checked out to User ID# {user_id}.")
+
+    elif choice=='12':
+        print(f"{Style.BRIGHT}\nChecking in? Whatcha got?")
+        user_id=input('User ID: ')
+        video_id=input('Video ID: ')
+        while video_id.isalpha() or user_id.isalpha() or not video_id or not user_id:
+            print(f"{Fore.RED}User ID or Video ID is not valid. Please enter a valid number.")
+            user_id=input('User ID: ')
+            video_id=input('Video ID: ')
+        if user_id.isnumeric():
+            user_id = int(user_id)
+        video_id=input('Video ID: ')
+        if video_id.isnumeric():
+            video_id = int(video_id)
+        RentalAPI.checkin(user_id=user_id, video_id=video_id)
+        print(f"{Fore.GREEN}Video #{video_id} successfully checked in.")
```

Comment:
What if a user tries to checkin a video that isn't checked-in. I'm unable to see your RentalAPI file, but it seems that this situation is not handled.